src/SourceBuild/content/repo-projects/winforms.proj

```diff
@@ -8,7 +8,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <RepositoryReference Include="arcade" />
```

Comment:
Nit: add back.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/608e3adab65aca27adcc828a17383c6b401466dc